starcheck/calc_ccd_temps.py

```diff
@@ -121,9 +122,10 @@ def get_ccd_temps(oflsdir, outdir='out',
     if not os.path.exists(outdir):
         os.mkdir(outdir)
 
-    module_dir = Path(__file__).parent
     if model_spec is None:
-        model_spec = str(module_dir / 'data' / 'aca_spec.json')
+        model_spec, chandra_models_version = get_xija_model_spec(
+            'aca',
+            version=proseco_char.chandra_models_version)
```

Comment:
I don't think you need this reference to the `proseco_char` version. Just doing `get_xija_model_spec('aca')` is going to get the latest flight version. If there is a need to be able to run this with a different version then it might be appropriate to put in a `chandra_models_version` kwarg and maybe even a starcheck command line arg.

Did you have some workflow in mind that required the `proseco_char` involvement?

Reply:
I didn't really have a workflow in mind, I just thought that being explicit here meant that I definitely wouldn't need to check that `chandra_models_version == proseco_char.chandra_models_version` .

Reply:
I've just cut the explicit reference.